### PR TITLE
feat: supports Chinese language

### LINKED ISSUE #111: Feature Request: Support none break words language like Chinese and Japenese
image

### PR DESCRIPTION
**Pull request by lbenie**
BREAKING CHANGES:
  - removes Options and Readingtime exported types
  - readingTime function now takes 3 parameters
    data --> string
    wordsPerMinute --> number
    language --> SupportedLanguages
  - complete rewrite of the underlyind methods

closes #111

### CODE REVIEW

File: lib/i18n/cn.ts [hunk 1]
```diff
@@ -0,0 +1,6 @@
+import { I18n } from './II18n'
+
+export const cn: I18n = {
+  less: '不到一分钟的阅读',
+  default: '阅读时间',
```
Comment (lbenie) [inline lib/i18n/cn.ts:5]: @Mister-Hope could you validate this for me.

This is straight from google translate 😓

Comment (Mister-Hope) [reply lib/i18n/cn.ts:5]: Emmm, in our habit we actually don't mention `reading` in the end. minute is `分钟` in Chinese.

Just
```
  less: '小于一分钟',
  default: '分钟',
```

will be fine. We can understaand it's reading time. E.g.: https://blog.dreace.top/

image

We wound more prefer `大约 xx 分钟` which is `about xx minute` in English

Comment (lbenie) [reply lib/i18n/cn.ts:5]: I will do a release later today thanks for helping

Gotta go back to work I have some meetings coming up

**Pull request merged**